internal/storage/storage.go

```diff
@@ -0,0 +1,177 @@
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/flynntdev/go_final_project/api"
+	"github.com/flynntdev/go_final_project/config"
+	"github.com/flynntdev/go_final_project/internal/task"
+)
+
+const (
+	limit = 20
+)
+
+type Store struct {
+	db *sql.DB
+}
+
+func NewStore(db *sql.DB) Store {
+	return Store{db: db}
+}
+
+func (s *Store) PostTask(t task.Task) (string, error) {
```

Comment:
Уффф, айди же это инт, кому нужно пусть сам конвертирует
```suggestion
func (s *Store) PostTask(t task.Task) (int, error) {
```